### DISCUSSION TITLE
unocss and SvelteKit, anyone got it working?

### DISCUSSION THREAD

**Discussion by lkj4** [sveltejs/kit#2873] [Q&A]
Pretty much the title and related thread with my attempt at the thread's bottom is here: https://github.com/antfu/unocss/issues/6

So, I just can't get it to work with SvelteKit, I guess must do something wrong. @aewing got it working but he seems to be not available atm, so I ask here again.

**Comment by aewing** [comment]
I have replied to your comments in the thread on the unocss repository - https://github.com/antfu/unocss/issues/6#issuecomment-977465990

**Reply by lkj4** [reply]
Thanks, it works like a charm! 🙂
